src/renderers/dom/client/ReactEventListener.js

```diff
@@ -133,6 +135,15 @@ function handleTopLevelWithPath(bookKeeping) {
       }
     }
   }
+  if (eventsFired === 0) {
```

Comment:
This does seem to be in line with what we do in the non-path case. Let's do it.

(tangent) I'd like to revisit some of this. I don't think the path case behaves the same as the non-path case in other ways, namely it looks like we fire events as we traverse here. The non-path case builds up the ancestors array, then fires them all (there's a comment about why, which sounds right). We can probably reduce the complexity a bit here and instead of having 2 functions that fire events, have the 2 functions collect the nodes that need to have the event fired, then fire them in the same place. Basically the `bookkeeping` object should be identical between browsers.